Classes/Service/ContentService.php

```diff
@@ -156,12 +156,11 @@ public function moveRecord(array &$row, &$relativeTo, $parameters, DataHandler $
         // This condition matches colPos values which are above the constant which is a virtual, assumed limit. When
         // the $relativeTo value is above this constant value it is likely a virtual value - but it may also be the
         // case for very large page trees where UID values naturally exceed this.
-        if (MiscellaneousUtility::UNIQUE_INTEGER_OVERHEAD < $relativeTo) {
+        if (MiscellaneousUtility::UNIQUE_INTEGER_OVERHEAD < $row['colPos']) {
```

Comment:
Note from TceMain applies here as well - these functions shouldn't be using `$row['colPos']` - that's what `$relativeTo` is there for.

Reply:
If you move an element to the first position, `$relativeTo` is always 1.
If you move an element after an other element, `$relativeTo` is a record uid of this other element.
I think we cannot use `$relativeTo` to extract the virtual colpos value.

Reply:
`$relativeTo` should not always be `1` - it can be either a negative or positive integer, if it's negative it's the UID of the element to move the record after, if it's positive it's supposed to be the colPos. Check the PHPDOC for the method ;)

Reply:
Ohh :) thanx ;)